packages/web/src/components/layouts/PackagePage/PackagePage.tsx

```diff
@@ -0,0 +1,596 @@
+
+              {/* TODO: there are no skeletons for vulnerabilities in design,
+                        not sure if intended or overlooked */}
+              {!loading && (
+                <section>
+                  <h2>Vulnerabilities</h2>
+
+                  <div className={styles.vulnerabilities}>
+                    {vulnerabilities.map(({ id, label, title, text, linkText, linkPath }) => (
+                      <div key={id} className={styles.vulnerability}>
+                        <div className={styles.vulnerabilityTop}>
+                          <Chip
+                            variant='vulnerabilities'
+                            size='badge'
+                            fontWeight='semiBold'
+                            icon={
+                              <Icon kind='vulnerability' width={24} height={24} color='white' />
+                            }
+                          >
+                            {label}
+                          </Chip>
+
+                          <a
+                            href={linkPath}
+                            target='_blank'
+                            rel='noreferrer'
+                            className={styles.vulnerabilityLink}
+                          >
+                            {linkText}
+                          </a>
+                        </div>
+                        <div className={styles.vulnerabilityTitle}>{title}</div>
+                        <div className={styles.vulnerabilityText}>{text}</div>
+                      </div>
+                    ))}
+                  </div>
+                </section>
+              )}
+            </div>
+
+            <aside className={styles.sidebar}>
+              <h4 className={styles.sidebarOptionalTitle}>Background information</h4>
+
+              <div className={styles.sidebarInner}>
+                <div className={styles.sidebarItem}>
+                  {/* TODO: not sure if making this title hidden for desktop is intentional */}
+                  <div className={clsx(styles.sidebarItemTitle, styles.sidebarItemTitleDesktop)}>
+                    Links
+                  </div>
+
+                  <div className={styles.sidebarLinks}>
+                    {loading ? (
+                      <>
+                        <Skeleton width={32} />
+                        <span className={styles.sidebarLinkSkeleton}>
+                          <Skeleton width={18} className={styles.sidebarLinkIconSkeleton} />
+                          <Skeleton className={styles.sidebarLinkTextSkeleton} />
+                        </span>
+                        <span className={styles.sidebarLinkSkeleton}>
+                          <Skeleton width={18} className={styles.sidebarLinkIconSkeleton} />
+                          <Skeleton className={styles.sidebarLinkTextSkeleton} />
+                        </span>
+                      </>
+                    ) : (
+                      <>
+                        <a
```

Comment:
Same here about moving data to `loadedData`. You can use `PackagePreview` component as a reference for input data format, as it's highly likely that format will be similar here.